gradle/versions.gradle

```diff
@@ -76,7 +76,7 @@ dependencyManagement {
 
     dependency 'org.zeroturnaround:zt-exec:1.11'
 
-    dependency 'org.hyperledger.besu:plugin-api:1.4.0'
-    dependency 'org.hyperledger.besu.internal:metrics-core:1.4.0'
+    dependency 'org.hyperledger.besu:plugin-api:21.1.1'
```

Comment:
nit: could use a dependencySet so the version only needs to be set in one place

Reply:
the package names are different ... besu vs besu.internal